objgraph.py

```diff
@@ -691,6 +695,7 @@ class name part.
         >>> show_backrefs(obj, filter=lambda x: not inspect.isclass(x))
         >>> show_backrefs(obj, highlight=inspect.isclass)
         >>> show_backrefs(obj, extra_ignore=[id(locals())])
+        >>> show_refs(obj, extra_node_attrs=lambda x: dict(URL=str(id(x))))
```

Comment:
This is the docstring for `show_backrefs()` so it's a bit weird to see an example calling a different function.

Reply:
Stupid copy&paste ;-)